dashboard/test/ui/features/hour_of_code/hour_of_code.feature

```diff
@@ -14,14 +14,17 @@ Scenario: Solving puzzle 1, proceeding to puzzle 2, verifying that puzzle 1 appe
   Then I close the dialog
   And I wait until I am on "http://studio.code.org/hoc/2"
   And I wait for the page to fully load
+  And I wait until jQuery Ajax requests are finished
```

Comment:
Why do we need these new waits?  (I'm removing most of these in my change and want to make sure I've covered the cases where we need this.)

Reply:
as we discussed last week, i was not thinking through which scenarios were using async progress fetching vs. having progress on page load, so i just started adding them everywhere we are verifying progress with a "might help, can't hurt" mentality. but per our conversation, i am expecting you to remove most of them in favor of the work you're doing. my build was green at that point so i didn't want to remove them from this PR. if that's just going to make more work for you though, i can.

Reply:
Oh right, thanks for the reminder.  No need to make changes to your PR, I'll modify after you merge, thanks!